test/posts.test.js

```diff
@@ -1,8 +1,17 @@
+const crypto = require('crypto');
+const normalizeUrl = require('normalize-url');
 const request = require('supertest');
 const app = require('../src/backend/web/app');
 const Post = require('../src/backend/post');
 const { addPost } = require('../src/backend/utils/storage');
 
+const encodeKey = url => {
```

Comment:
In a follow-up, I'm going to move this into the logic of the `Post` and `Feed` classes, so it's not something we have to do externally (i.e., it should happen automatically for us).  For now, this is fine.